code/__DEFINES/actionspeed_modification.dm

```diff
@@ -0,0 +1,2 @@
+//ids
+#define ACTIONSPEED_ID_SANITY					"sanity_component"
```

Comment:
it's super common in the codebase and other maintainers yell at you for not doing it. might wanna get a consensus on it. in this case it doesnt matter because theres only one entry tho